### ISSUE TITLE
Invalid handle error on Windows

### ISSUE THREAD

**Issue by porty** [99designs/aws-vault#112]
# Problem

When entering access credentials, `aws-vault` exits with the error message "The handle is invalid"

```
C:\Users\shorty>aws-vault add test
Enter Access Key ID: AKIAAAAAAAAAAAAAAAAAAAAA
Enter Secret Access Key: zzzzzzzzzzzzzzzzzzzzzzz
Enter passphrase to unlock C:\Users\shorty\.awsvault\keys: aws-vault: error: The handle is invalid.
```

# Expected Result

aws-vault accepts password input and exists successfully.

# Environment Details

OS: Windows 10 x64 build 15019
Go version: 1.7.3
Built from tag v3.7.0
Using standard Windows Command prompt

**Comment by lox** [comment]
Did you fix this in #113 @porty?

**Comment by porty** [comment]
Yep, said so up the top

> Fix for #112

**Comment by zhkostadinov** [comment]
The issue is still there on Windows when you try to do that with bash windows. 
The workaround is to use PowerShell with admin privileges.
